src/PowerShellEditorServices/Services/DebugAdapter/DebugEventHandlerService.cs

```diff
@@ -117,59 +118,47 @@ private void PowerShellContext_DebuggerResumed(object sender, DebuggerResumeActi
             _debugAdapterServer.SendNotification(EventNames.Continued,
                 new ContinuedEvent
                 {
-                    AllThreadsContinued = true,
-                    ThreadId = 1
+                    ThreadId = 1,
+                    AllThreadsContinued = true
                 });
         }
 
         private void DebugService_BreakpointUpdated(object sender, BreakpointUpdatedEventArgs e)
         {
-            string reason = "changed";
-
+            // Don't send breakpoint update notifications when setting
+            // breakpoints on behalf of the client.
             if (_debugStateService.IsSetBreakpointInProgress)
             {
-                // Don't send breakpoint update notifications when setting
-                // breakpoints on behalf of the client.
                 return;
             }
 
-            switch (e.UpdateType)
-            {
-                case BreakpointUpdateType.Set:
-                    reason = "new";
-                    break;
-
-                case BreakpointUpdateType.Removed:
-                    reason = "removed";
-                    break;
-            }
-
-            var breakpoint = new OmniSharp.Extensions.DebugAdapter.Protocol.Models.Breakpoint
-            {
-                Verified = e.UpdateType != BreakpointUpdateType.Disabled
-            };
-
             if (e.Breakpoint is LineBreakpoint)
             {
-                breakpoint = LspDebugUtils.CreateBreakpoint(BreakpointDetails.Create(e.Breakpoint));
+                var breakpoint = LspDebugUtils.CreateBreakpoint(
+                    BreakpointDetails.Create(e.Breakpoint, e.UpdateType)
+                );
+
+                string reason = (e.UpdateType) switch {
+                    BreakpointUpdateType.Set => "new",
+                    BreakpointUpdateType.Removed => "removed",
+                    BreakpointUpdateType.Enabled => "changed",
+                    BreakpointUpdateType.Disabled => "changed",
```

Comment:
[We can improve on the strings here too](https://github.com/OmniSharp/csharp-language-server-protocol/blob/1b6788df2600083c28811913a221ccac7b1d72c9/src/Dap.Protocol/Feature/Events/BreakpointFeature.cs#L103-L109)

Reply:
Good call.